hips/tiles/tile.py

```diff
@@ -0,0 +1,131 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
```

Comment:
You have this import line in `hips/tiles/tile.py`.
Yes, you need it in tests files.
But no, you don't need it in `hips/tiles/tile.py`.